gui/slick/interfaces/default/config_postProcessing.tmpl

```diff
@@ -123,7 +123,15 @@
 						    <label class="nocheck">
 						        <span class="component-title">&nbsp;</span>
 						        <span class="component-desc"><b>NOTE:</b> Scripts are called after SickRage's own post-processing.</span>
-						    </label>						    
+						        <span class="component-desc"> See <a href="https://github.com/SiCKRAGETV/SickRage/wiki/Post%20Processing"><font color='red'><b>Wiki</b></font></a> for enviroment variables passed to the script.</span>
+						    </label>
+						    #if sys.platform == 'win32' 
```

Comment:
Instead of making this windows only, we could simply say:
Note: For any scripted languages, include the interpreter executable before the script(e.g.: c:\python27\python.exe d:\script.py)